docs/user/openstack/privileges.md

```diff
@@ -0,0 +1,51 @@
+# Required Privileges
+
+In order to install an OpenShift cluster on OpenStack successfully, the user provided to the installer needs certian privileges. The easiest way to achieve this level of permission and ensure success is to install with a user who has administrative privileges.
```

Comment:
That's a bit like saying the easiest way to run a program on a *nix system is to run as root.  Since ShiftOnStack deployments are intended to be usable by regular users without any special privileges I'd advocate for a document that notes default user privileges as such and any needed departures from these.  Each departure represents a service ticket the user must open to the OSP administrator (think "help" ticket to PSI) rather than self-service IAAS that OSP is supposed to provide regular users.

Reply:
Do you know where I can find better documentation about what permissions a default openstack user has @tombarron ? I have been having a hard time finding that information

Reply:
I agree, we should recommend not to use administrative privileged!

Reply:
I can't tell you all the default permissions for OpenStack because there are so  many different OpenStack components but in general when an OpenStack user is created it is added to a Keystone "project" and the user has permission to create, list, show, update, and delete OpenStack resources in the context of that project -- Nova VMs, Neutron networks and routers, floating IPs, glance images, cinder volumes, manila shares, and  so on.
These are the ordinary permissions that a user gets on PSI without asking for extras.  IIRC, the only extra permission needed (that would require a ticket to the PSI admins) was the Swift operator role and that is no longer required either?

Reply:
Perhaps the best way to capture the normal user default permissions is in terms of the "member" role as explained here [1] and here [2].  An ordinary user with the "member" role rather than "admin" role should be able to install an OCP cluster on OpenStack in the context of their own keystone project.  That way there can be hundreds of independent, unrelated OCP clusters belonging to different regular users.  OpenStack is a "private cloud" version of Infrastructure As A Service -- more like AWS or GCP than virtualization platforms like RHEV and vSphere where a regular user without additional privileges cannot create VMs or other required resources.

[1] https://docs.openstack.org/keystone/latest/admin/service-api-protection.html
[2] https://wiki.openstack.org/wiki/User_Management

Reply:
Thanks for all the info, its very helpful. I will restructure this section to steer customers away from using admin users and to define what permissions the member level account will need more clearly.

Reply:
Well thank you for understanding, and for this work!

Reply:
yeah! I agree! It is better to explain the priviledge for OpenStack in the OpenStack way, so saying "member" role rather than "all" is way more accurate